packages/nx/src/tasks-runner/run-command.ts

```diff
@@ -194,7 +194,7 @@ export async function runCommand(
 }
 
 function setEnvVarsBasedOnArgs(nxArgs: NxArgs, loadDotEnvFiles: boolean) {
-  if (nxArgs.outputStyle == 'stream' || process.env.NX_BATCH_MODE === 'true') {
```

Comment:
It seems like this functionality is lost now - previously batch mode meant streamed logs, now it doesn't. I wonder if there's a good way to check if any task will use batch mode

Reply:
We might be able to only set stream when we actually run the batch.